backend/account/tests/test_views.py

```diff
@@ -146,6 +146,79 @@ def test_invalid_oauth_properties(self, mock_get_or_create_user):
         mock_get_or_create_user.assert_not_called()
 
 
+class PrivacyTests(TestCase):
+    def setUp(self):
```

Comment:
did you uhhhh delete my comment where I said I dont think the csrf stuff is required for privacy tests 😳